packages/block-library/src/video/edit-common-settings.js

```diff
@@ -83,6 +83,7 @@ const VideoSettings = ( { setAttributes, attributes } ) => {
 				) }
 			/>
 			<SelectControl
+				__next40pxDefaultSize
```

Comment:
This is the "Preload" dropdown in the Video block inspector controls. (It's named like it's a reused component, but currently only used in the Video block.)